util/reggen/countermeasure.py

```diff
@@ -2,25 +2,54 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
+import re
 from typing import Dict, List
 
 from .lib import check_keys, check_str, check_list
 
-# TO BE DEFINED
 CM_ASSETS = [
-    'FOO'
+    'KEY',
```

Comment:
nice! 
in case the intent of these names aren't clear, where do you think those should be documented?

Reply:
I would say we add it to the comportable IP spec or similar...

Reply:
We can do that once the D2S checklist is finalized.

Reply:
that makes sense.